src/raftstore/store/snap.rs

```diff
@@ -924,6 +926,7 @@ impl Snapshot for Snap {
             {
                 let mut file = cf_file.file.take().unwrap();
                 file.flush()?;
+                file.sync_all()?;
```

Comment:
If the size is 0 like L920, will there be any file generated?

Reply:
Seems no, see L459